kahuna/public/js/directives/gr-file-change.js

```diff
@@ -0,0 +1,15 @@
+import angular from 'angular';
+
+export var fileChange = angular.module('gr.fileChange', []);
+
+fileChange.directive('grFileChange', ['$parse', function($parse) {
+    return {
+        restrict: 'A',
+        link: function(scope, element, attrs) {
+            element.on('change', function() {
+                const onchange = $parse(attrs.grFileChange)(scope);
+                onchange(Array.from(element[0].files));
```

Comment:
One pattern I've noticed in other Angular libs is rather than passing "function pointers" to directives, they expose data to expressions and evaluate the expressions.

e.g. instead of 
``` html
<input gr-file-change="doSomething">
```

do

``` html
<input gr-file-change="doSomething($files)">
```

by invoking

``` js
scope.$eval(attrs.grFileChange, {$files: Array.from(element[0].files)})
```

Thoughts?